src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/mgr-summary.pipe.ts

```diff
@@ -0,0 +1,34 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+import _ from 'lodash';
+
+@Pipe({
+  name: 'mgrSummary'
+})
+export class MgrSummaryPipe implements PipeTransform {
+  transform(value: any): any {
+    if (!value) {
+      return '';
+    }
+
+    let activeCount = $localize`n/a`;
+    const activeTitleText = _.isUndefined(value.active_name)
+      ? ''
+      : `${$localize`active daemon`}: ${value.active_name}`;
+    // There is always one standbyreplay to replace active daemon, if active one is down
+    if (activeTitleText.length > 0) {
+      activeCount = '1';
+    }
+
+    const mgrSummary = [
+      {
+        // Displays Active Manager
+        content: `${activeCount}`,
+        class: 'popover-info',
+        titleText: activeTitleText
+      }
+    ];
```

Comment:
Pipes are mainly for transforming plain text. Something that both handles text but also presentation (that CSS class) and structutural (title and content) concerns should rather be an isolated component.

Reply:
Sounds good, I think the title and the style class passed through when referencing the old pipes. Now the pipe only returns the content :+1: